.babelrc

```diff
@@ -1,5 +1,8 @@
 {
-  "presets": [],
+  "presets": [
+    "es2015",
```

Comment:
Can we try and isolate the issues requiring this tranpilation to be required? 1. So that we can keep an eye on what needs to be updated in their codebase so that we can remove these and 2. So we can remove any that aren't necessary.

Reply:
This is for the regeneratorRuntime polyfill.

Reply:
Ahh ok, this is how I resolved it in the past: 

`parcel-bundler/parcel/issues/916#issuecomment-419154118`

Add this to the package.json
```json
  "browserslist": [
    "last 1 Chrome version"
  ]
```

Reply:
This does work for running parcel, but parcel build now dies with `Maximum call stack size exceeded` in `...node_modules/parcel-bundler/src/utils/objectHash.js`. It must be causing something to be unhappy. No idea where though.